index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <h1 id="header">
+      <meta charset="UTF-8">
+      <title>Pair Programming Two</title>
+      <link rel="stylesheet" href="css/style.css">
+      <nav id="nav-bar" >
+        <input href="#header" class = "nav-link" type="button" onclick="checkName()" value="Check A" />
+        <input href="#header" class = "nav-link" type="button" onclick="checkName()" value="Check B" />
+        <input href="#header" class = "nav-link" type="button" onclick="checkName()" value="Check C" />
+      </nav>
+
+      <img src="https://cdn.logo.com/hotlink-ok/logo-social.png" id="header-img">
+
+      <video src="https://www.youtube.com/watch?v=qz0aGYrrlhU" id="video"></video>
```

Comment:
Image and Video in a header tag are not recommended. You can use CSS to add an image as a background.